CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [3.0.1] - 2023-09-13
+
+### Added
+
+- Funding files and updates in the Readme about funding the project.
+
+### Changed
+
+- The `WordPress.Security.EscapeOutput` sniff has more modular error codes, so that developers can ignore certain output escaping cases based on where the errors occur. Props [@anomiex].
```

Comment:
IMO this changelog entry needs to be made more specific and possibly moved to "Added" ?

We need to make it clear that it is only the new functionality added in 3.0.0 regarding flagging parameters passed to exceptions created in a throw statement which is affected + the new error code for those needs to be mentioned. It should also be mentioned that the error code of other things being flagged by the sniff has not changed.

Think of it this way: if you read this as a user of the EscapeOutput sniff, would you know what to do/what you could do after reading this line ?

Reply:
I will move it to added and explain it a bit more in details 👍🏼